cli/config/config.go

```diff
@@ -142,18 +142,82 @@ func InitConfig() {
 	newConfig(w)
 }
 
-func checkConfigDirectory(dir string) {
-	found, err := exists(dir)
-	if found && err == nil {
-		return
+// PutServiceEnvironment puts the environment variables for the service, replacing "SERVICE_"
+// with service name in uppercase. The variables are:
+//  - SERVICE_VERSION: where it represents the version of the service (i.e. APACHE_VERSION)
+//  - SERVICE_PATH: where it represents the path to its compose file (i.e. APACHE_PATH)
```

Comment:
Follow-up: we won't need the *_PATH after updating the fetched compose files to remove the `build` section.